tests/samplers_tests/test_nsgaii.py

```diff
@@ -146,6 +148,125 @@ def constraints_func(trial: FrozenTrial) -> Sequence[float]:
         assert trial.system_attrs[_CONSTRAINTS_KEY] == (trial.number,)
 
 
+@pytest.mark.parametrize("dir1", [StudyDirection.MINIMIZE, StudyDirection.MAXIMIZE])
+@pytest.mark.parametrize("dir2", [StudyDirection.MINIMIZE, StudyDirection.MAXIMIZE])
+@pytest.mark.parametrize(
+    "constraints_list",
+    [
+        [[]],  # empty constraint
+        [[-float("inf")], [-1], [0]],  # 1d constraints
+        [[c1, c2] for c1 in [-float("inf"), -1, 0] for c2 in [-float("inf"), -1, 0]],
+    ],
+)  # 2d-constraints
```

Comment:
This comment should be placed in 2 line up.

Reply:
Fixed. Thanks!